addons/account/static/src/components/account_type_selection/account_type_selection.js

```diff
@@ -16,7 +16,7 @@ export class AccountTypeSelection extends SelectionField {
             { name: _t('Profit & Loss') },
             { name: _t('Income'), children: opts.filter(x => x[0] && x[0].startsWith('income')) },
             { name: _t('Expense'), children: opts.filter(x => x[0] && x[0].startsWith('expense')) },
-            { name: _t('Other'), children: opts.filter(x => x[0] && x[0] === 'off_balance') },
+            { name: _t('Other'), children: opts.filter(x => x[0] && x[0] === 'off') },
```

Comment:
This component is used for the account_type field, and will have the `'off_balance'` option